crates/planner/src/types.rs

```diff
@@ -209,7 +209,9 @@ impl<'a> RootGroup<'a> for QueryRootGroup<'a> {
     }
 
     fn into_selection_set(self) -> Vec<(&'a str, SelectionRefSet<'a>)> {
-        self.0.into_iter().collect()
+        let mut services: Vec<(&'a str, SelectionRefSet<'a>)> = self.0.into_iter().collect();
+        services.sort_by(|(a, _), (b, _)| a.cmp(b));
```

Comment:
I think if sorted list of services is needed, better to use sorted collection for it and not rely on Vector. But maybe it's not worth it.

Reply:
Introducing list sorting to make the results as deterministic as possible.